README.md

```diff
@@ -4,33 +4,64 @@ Software Engineering Day1 Assignment
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
-
+Software engineering is the application of engineering principles to design,develop,test and deploy softwate systems.
+It helps to create software application in various aspects of modern life,such as commerce,communication,entertainmentvand health care.
 
```

Comment:
1. The development of high-level programming languages like Fortran (1957), COBOL (1959), and later C (1972) marked a significant shift from machine and assembly languages. These languages allowed programmers to write code that was more abstract, readable, and easier to manage, significantly improving productivity and enabling the development of more complex software. 
2. The introduction of Structured programming, popularized by languages like Pascal (1970) and the work of Edsger Dijkstra, emphasized breaking programs into modular, reusable blocks of code and avoiding the use of "goto" statements.This approach led to more reliable and maintainable code, laying the groundwork for modern software engineering practices, including object-oriented programming and agile methodologies.
3. The Agile Manifesto (2001) introduced a set of principles promoting iterative development, collaboration, and responsiveness to change, challenging traditional, rigid software development practices like the Waterfall model.Agile methodologies revolutionized software development by enabling teams to deliver software more frequently, with a focus on customer satisfaction, adaptability, and continuous improvement, which remains a dominant approach in the industry today.